pkg/manager/manager.go

```diff
@@ -289,17 +290,82 @@ func (m *manager) AttachRepresentor(conf *types.PluginConf) error {
 		if err = utils.BridgeTrunkVlanAdd(m.nLink, rep, conf.Trunk); err != nil {
 			return fmt.Errorf("failed to add trunk VLAN for representor %s: %v", conf.Representor, err)
 		}
+		log.Info().Msgf("Setting multiple VLANs for rep %s: %v", conf.Representor, conf.Trunk)
 	}
 
 	if conf.Vlan > 0 {
 		if err = utils.BridgePVIDVlanAdd(m.nLink, rep, conf.Vlan); err != nil {
 			return fmt.Errorf("failed to set VLAN for representor %s: %v", conf.Representor, err)
 		}
+		log.Info().Msgf("Setting PVID VLAN for rep %s: %d", conf.Representor, conf.Vlan)
+	}
+
+	// add vlan config to uplink if configured to do so
```

Comment:
nit: this bit is pretty straight forward (conditionally add vlans to uplink) so no need for comment.